outport/process/alteredaccounts/alteredAccountsProvider_test.go

```diff
@@ -517,6 +521,7 @@ func testExtractAlteredAccountsFromPoolShouldNotIncludeReceiverAddressIfNftCreat
 		},
 	}
 	args := getMockArgs()
+	args.AddressConverter = testscommon.NewPubkeyConverterMock(26)
```

Comment:
If this is the len of `"sender in shard 0 - tx 1 "`, then please use it as len(variableName)

Reply:
done